amundsen_application/static/js/components/DashboardPage/ImagePreview/index.tsx

```diff
@@ -14,17 +15,43 @@ export interface ImagePreviewProps {
 interface ImagePreviewState {
   isLoading: boolean;
   hasError: boolean;
+  isModalVisible: boolean;
 }
 
-export class ImagePreview extends React.Component<ImagePreviewProps, ImagePreviewState> {
-  constructor(props) {
-    super(props);
+type PreviewModalProps = {
+  imageSrc: string,
+  onClose(): void,
```

Comment:
So this is just another way of doing: `onClose: () => void`?

Reply:
Oh, it seems!
I was googling fast for a way of making the warning go. I can put your proposed one there if it is more starndard.

Reply:
I can't claim its more standard because I don't know if that's actually true. Consistency would help for people who are newer the space and might not immediately know this is just another way of doing the same thing, but don't mind this as is.